common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -6,9 +6,8 @@
 
 import json
 import logging
-import warnings
-
 from lxml import etree
+import warnings
```

Comment:
Oh - no - @rlucioni in your original PR, you moved the lxml import to above warnings. This diff is just displaying differently: https://github.com/edx/edx-platform/pull/10261/files#diff-14daf51ec8decc308953f7a4e7e0a03fL8

Can you advise? Where should the import of lxml go?